http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -1144,4 +1155,12 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
     immutable.Map(googleKey -> startupScriptContent)
   }
 
+  private def isClusterBeforeCutoffDate(cluster: Cluster): Boolean = {
+    (for {
+      dateStr <- dataprocConfig.clusterCutoffDate
+      date <- Try(new SimpleDateFormat("yyyy-MM-dd").parse(dateStr)).toOption
+      isClusterBeforeCutoffDate = cluster.auditInfo.createdDate.isBefore(date.toInstant)
+    } yield isClusterBeforeCutoffDate) getOrElse false
```

Comment:
if we can't get cluster's created date, it's probably in bad state? if that's the case, shall we default to `true`?

Reply:
If the config is not present or it's not parseable we probably shouldn't block cluster start for all clusters... hence defaulting to `false`.